containers/Readme.md

```diff
@@ -79,3 +79,15 @@ export KUBERNETES_CLIENT_CERTIFICATE_FILE=$OPENSHIFT_HOME/openshift.local.config
 export KUBERNETES_CLIENT_KEY_FILE=$OPENSHIFT_HOME/openshift.local.config/master/admin.key
 mvn clean install -Pcontainer-tests
 
+Deploying the Docker Images to a Registry
```

Comment:
I've been able to run the container tests successfully this morning. So I've updated the doc. Also I've converted it to Asciidoc which is the standard on all H* projects. After you've fixed the pluginMgmt section issues I'll merge this PR manually and will convert this last paragraph to Asciidoc.